src/validators/expenses.validator.js

```diff
@@ -0,0 +1,35 @@
+const { sendErrorResponse } = require('../helpers/sendErrorMessage');
+const { STATUS_CODES, ERRORS } = require('../variables/variables');
+
+const validateId = (id, res) => {
+  if (!id) {
+    sendErrorResponse(res, STATUS_CODES.BAD_REQUEST, ERRORS.ID_REQUIRED);
+
+    return false;
+  }
+
+  return true;
+};
+
+const validateExpenses = (expenses, res) => {
+  if (!expenses) {
```

Comment:
Validation functions shouldn't know anything about response and what to send, just validate data and throw errors if needed